spec/items_repo_spec.rb

```diff
@@ -1,12 +1,161 @@
 require 'rspec'
+require 'bigdecimal'
 require './lib/sales_engine'
 require './lib/items'
 require './lib/items_repo'
 require './lib/merchants'
 require './lib/merchants_repo'
 
+
 RSpec.describe ItemRepo do
 
+  se = SalesEngine.from_csv({
```

Comment:
All test look great, with the refactor. Think we are in a great spot to move on to our next iteration